kedro-datasets/kedro_datasets/ibis/table_dataset.py

```diff
@@ -126,6 +127,9 @@ def __init__(  # noqa: PLR0913
                 in a multi-level table hierarchy.
             connection: Configuration for connecting to an Ibis backend.
                 If not provided, connect to DuckDB in in-memory mode.
+            credentials: Connection information (e.g.
+                user, password, token, account). If provided, these values
+                overlay the base `connection` configuration.
```

Comment:
```suggestion
                override the base `connection` configuration.
```
I think is the correct word here?